docs/user/resources/02-logpipeline.md

```diff
@@ -118,6 +118,7 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
 | **output.&#x200b;http.&#x200b;host.&#x200b;valueFrom.&#x200b;secretKeyRef.&#x200b;key**  | string | The name of the attribute of the Secret holding the referenced value. |
 | **output.&#x200b;http.&#x200b;host.&#x200b;valueFrom.&#x200b;secretKeyRef.&#x200b;name**  | string | The name of the Secret containing the referenced value |
 | **output.&#x200b;http.&#x200b;host.&#x200b;valueFrom.&#x200b;secretKeyRef.&#x200b;namespace**  | string | The name of the Namespace containing the Secret with the referenced value. |
+| **output.&#x200b;http.&#x200b;keepBody**  | boolean | If `false`, after successfully parsing the field `log` is removed from the log record. Default is `true`. |
```

Comment:
It is not very clear which field gets parsed and which gets dropped, maybe:
```suggestion
| **output.&#x200b;http.&#x200b;keepBody**  | boolean | If the log payload contained in the `log` attribute was in JSON and was parsed successfully, the `log` attribute will not be dropped if `keepBody` is `true`, otherwise it gets removed from the log record. Default is `true`. |
```

Reply:
Attribute description updated